plugins/quetz_tos/quetz_tos/api.py

```diff
@@ -33,11 +33,15 @@ def post_file(file):
     return file.filename
 
 
-@router.get("/api/tos", tags=['Terms of Service'])
-def get_current_tos(db: Session = Depends(get_db)):
+@router.get("/api/tos/?{lang}", tags=['Terms of Service'])
+def get_current_tos(lang: str, db: Session = Depends(get_db)):
+    #lang = request.args.get('language')
+
     current_tos = (
-        db.query(TermsOfService).order_by(TermsOfService.time_created.desc()).first()
+        #db.query(TermsOfService).order_by(TermsOfService.time_created.desc()).first()
+        db.query(TermsOfService).filter_by(language == f"{lang}").first()
```

Comment:
Does it work? It seems `language` is not defined here?

For the code styling, `f"{lang}"` can simply be `lang`.

Maybe the following works? (fetch all terms of services and get the first one that matches the language)
```python
terms_of_services = (
    db.query(TermsOfService).order_by(TermsOfService.time_created.desc()).all()
)

current_tos = None
for tos in terms_of_services:
    if tos.language == lang:
        current_tos = tos
```